site/src/Librecores/ProjectRepoBundle/Entity/Project.php

```diff
@@ -312,6 +312,14 @@ class Project
      */
     private $releases;
 
+    /**
+     * Code quality metrics for the project
+     * @var float
+     *
+     * @ORM\Column(type="float", nullable=true)
```

Comment:
What's the motivation for storing this as float? Isn't a (possibly scaled, e.g. multiplied by 10) integer sufficient?

Reply:
@imphil The function getCodeQualityScore that calculates the project quality metrics returns a float value according to its definition

Reply:
Floats are really tricky to get right in comparisons (and hence sorting), integers are much easier to handle. How about multiplying the metrics value by 100 and storing that as integer value, effectively limiting the precision of the float to two digits.